Source/DafnyCore/AST/SourceOrigin.cs

```diff
@@ -8,6 +8,7 @@
 namespace Microsoft.Dafny;
 
 [method: SyntaxConstructor]
+[SyntaxBaseType(typeof(IOrigin))]
```

Comment:
nit: Isn't this redundant?

Reply:
No because IOrigin is an interface, so it's not a Type.BaseType